repo_resource/common.py

```diff
@@ -218,7 +218,8 @@ def __init__(self, url, revision='HEAD', name='default.xml',
         self.__name = name
```

Comment:
commit message. Please follow the commit message conventions done in this project.

When looking at the history for this file:
```
git log --oneline -- repo_resource/common.py
```

Shows:
```
common: <additional info>
```

Reply:
A suggestion: example:  `common: handle default element without revision`